### PR TITLE
[fix bug 1248893] Force curators for new groups.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: mozillians/groups/forms.py [hunk 1]
```diff
@@ -27,12 +27,13 @@ class GroupForm(happyforms.ModelForm):
...
         min_value=1,
         label='Membership will expire after',
-        required=False
     )
 
     def __init__(self, *args, **kwargs):
         super(GroupForm, self).__init__(*args, **kwargs)
         self.fields['curators'].required = False
+        if not self.instance.pk:
+            self.fields['curators'].required = True
```
Comment (johngian) [inline mozillians/groups/forms.py]: ``self.fields['curators']`` is also required when ``self.instance.pk`` and ``self.instance.groups.exists()``.

**Comment by johngian** [comment]
Please add a new test for that.

File: mozillians/groups/forms.py (hunk 1 above)
Comment (akatsoulas) [inline mozillians/groups/forms.py]: Sure, this is already handled in the clean method.

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
r+

**Pull request merged**